discord/enums.py

```diff
@@ -76,12 +76,19 @@ class EnumMeta(type):
         _enum_member_map_: ClassVar[Dict[str, Any]]
         _enum_value_map_: ClassVar[Dict[Any, Any]]
 
-    def __new__(cls, name, bases, attrs):
+    def __new__(cls, name, bases, attrs, **kwargs):
         value_mapping = {}
         member_mapping = {}
         member_names = []
 
         value_cls = _create_value_cls(name)
+
+        if kwargs.get('comparable'):
+            value_cls.__le__ = lambda self, other: isinstance(other, self.__class__) and self.value <= other.value
+            value_cls.__ge__ = lambda self, other: isinstance(other, self.__class__) and self.value >= other.value
+            value_cls.__lt__ = lambda self, other: isinstance(other, self.__class__) and self.value < other.value
+            value_cls.__gt__ = lambda self, other: isinstance(other, self.__class__) and self.value > other.value
```

Comment:
Move this to `_create_value_cls`. This stuff was in that function to prevent the unnecessary closure.